### DISCUSSION TITLE
プレゼントカレンダーのメールについて

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#932] [要望・アイデア・不具合]
日付が埋まっていないプレゼントカレンダーについて、タグ情報を元に過去関連する記事を投稿したユーザー宛に、参加を促すメールを送っているかと思います。

メールを送る自体は良いと思いますが、既に登録しているカレンダーでありながら、メールを送ってくるのはノイズですので、やめていただけると幸いです。また「xxxに関する深い見識をお持ちだと感じており〜」の文章も求人で送ってくるようなメールに似ていて、文章が不快に感じます。

つきましては、メール送信時に既に登録している参加者は除外するなどの対応をお願いします。

**Comment by ohakutsu** [comment]
@Brutus03 
ご意見ありがとうございます。

この度は、ご不快な思いをさせてしまい申し訳ございません :bow:
次回以降のメールから、既にカレンダーに参加登録されているユーザーを配信に含めないよう変更します。
また、メール文章についても改善できるよう検討いたします。

**Reply by Brutus03** [reply]
@ohakutsu 
恐縮ですが、背景隠して機械的な文章で送らてきたのが不快に感じました。

素直にカレンダーが埋まっていないことを伝える様にメールを送るのが良いと思います。
また、見せ方変えるのであれば「xxxに関する深い見識をお持ちだと感じており〜」の文章より、プレゼントがあることを前に出した方がユーザー的に食いつきが良いと思います。

**Reply by ohakutsu** [reply]
@Brutus03 
重ね重ね申し訳ございません :bow:
追加でいただいたご意見も踏まえて検討させていただきます！
